.travis.yml

```diff
@@ -6,7 +6,7 @@ python:
   - '3.4'
   - '3.5'
   - '3.6'
-  - pypy
+  - pypy-5.3.1
```

Comment:
Are tests failing on the latest pypi?

Reply:
They were. There's a mismatch between pypy and the cryptography module that I could only resolve by upgrading pypy from 2.5 (the default) to 5.3.1.